os/root-filesystem-placement.md

```diff
@@ -0,0 +1,64 @@
+# Configuring Root Filesystem Placement
+Container Linux supports composite disk devices such as RAID arrays. If the root filesystem is placed on a composite device, special care must be taken to ensure Container Linux can find and mount the filesystem early in the boot process. GPT partition entries have a [partition type GUID](https://en.wikipedia.org/wiki/GUID_Partition_Table#Partition_type_GUIDs) that specifies what type of partition it is (e.g. Linux filesystem). Container Linux uses special type GUIDs to indicate that a partition is a component of a composite device containing the root filesystem.
```

Comment:
The transition to the third sentence is abrupt.  Maybe join the third and fourth sentences with a semicolon, `, and`, etc., to make the flow clearer?